packages/gui/src/components/settings/ProfileView.tsx

```diff
@@ -23,6 +24,49 @@ const StyledCard = styled(Card)(({ theme }) => `
   background-color: ${theme.palette.background.paper};
 `);
 
+const InlineEdit = ({ text, walletId }) => {
+  const [editedText, setEditedText] = useState(text);
+  const [setDid, { isLoading: isSetDidLoading }] = useSetDIDNameMutation();
+
+  useEffect(() => {
+    setEditedText(text);
+  }, [text]);
+
+  const handleChange = (event) => setEditedText(event.target.value);
+
+  const handleKeyDown = (event) => {
+    if (event.key === "Enter" || event.key === "Escape") {
+      event.target.blur();
+    }
+  }
+
+  const handleBlur = (event) => {
+    if (event.target.value.trim() === "") {
+      setEditedText(text);
+    } else {
+      setDid({ walletId: walletId, name: event.target.value});
+    }
+  }
+
+  return (
+    <input
```

Comment:
The styling looks out of place using a plain input element. Does TextField work?

![image](https://user-images.githubusercontent.com/339312/171281244-7cc4cb0d-b12a-4bae-b23d-fce1bb9fa7d7.png)